test/unit/resolve.js

```diff
@@ -200,12 +229,5 @@ describe('resolve', function () {
         done()
       })
     })
-
-    it('should throw on unknown version spec', function (done) {
-      resolve('some-package', 'unknown spec', function (err) {
-        assert(err)
-        done()
-      })
-    })
```

Comment:
Now all other version strings are treated as tags. There is no need to test this case now.